test/models/concerns/gamelogic_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class GameLogicTest < ActiveSupport::TestCase
+  include GameLogic
+
+  fixtures :employees
+  attr_accessor :pairs, :names, :couples, :left, :year
+
+  def setup
+    @game = Game.new
+    @game.setting_employees
+  end
+
+  test 'The couples are created correctly' do
+    assert @game.couples[0].count >= 2
+  end
+
+  test 'the couples arent repeated' do
+    previous_games = [games(:game_one), games(:game_two), games(:game_three)]
+    previous_game_pairs = previous_games.map(&:couples)
+    @game.year = 2026
+    binding.break
```

Comment:
Uno de estos en producción y grave....

Van dos 😵‍💫 

```suggestion
    binding.break
```